src/main/java/asynchronous/typing/TypingTestTeam.java

```diff
@@ -0,0 +1,202 @@
+package asynchronous.typing;
+
+import java.util.Arrays;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
+
+import zyenyo.BotConfig;
+import zyenyo.Database;
+
+import commands.Typing;
+import dataStructures.TypingSubmission;
+import net.dv8tion.jda.api.EmbedBuilder;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+
+public class TypingTestTeam extends TypingTestTemplate
+{
+	private String[] teamRed;
+	private String[] teamBlue;
+	private final static short WPM_MINIMUM = 30;
+
+	public TypingTestTeam(MessageReceivedEvent event, String[] args)
+	{
+		super(event, args);
+		String[] players = Arrays.copyOfRange(args, 1, args.length);
+
+		teamRed = Arrays.copyOfRange(players,0, (int)players.length/2);
+		teamBlue = Arrays.copyOfRange(players, (int)players.length/2, players.length);
+	}
+
+	@Override
+	public void run() throws NumberFormatException
+	{
+		constructAndSendTest();
+		event.getJDA().addEventListener(this);
+	}
+	
+	
+	protected void constructAndSendTest() // Essentially a duplicate of 
+	{
```

Comment:
a duplicate of... ? XD

Reply:
I meant to say "Essentially a duplicate of TypingTest's constructAndSendTest." Looks like I got sidetracked. xD